### PR TITLE
fixes in global pooling with bilinear interpolation

### PR DESCRIPTION
**Pull request by RisaMagpie**
https://github.com/analysiscenter/batchflow/issues/561

### CODE REVIEW

File: batchflow/models/torch/layers/modules.py [hunk 1]
```diff
@@ -45,13 +45,9 @@ def __init__(self, inputs, layout='cna', filters=None, kernel_size=1, pool_op='m
...
                 pool_strides = tuple(np.floor((spatial_shape - 1) / level + 1).astype(np.int32).tolist())
 
-                layer = ConvBlock(inputs=x, layout='p' + layout, filters=filters, kernel_size=kernel_size,
-                                  pool_op=pool_op, pool_size=pool_size, pool_strides=pool_strides, **kwargs)
-                x = layer(x)
-
-                upsample_layer = Upsample(inputs=x, factor=None, layout='b',
-                                          shape=tuple(spatial_shape.tolist()), **kwargs)
-                module = nn.Sequential(layer, upsample_layer)
+                module = ConvBlock(inputs=x, layout='pb' + layout, filters=filters, kernel_size=kernel_size,
```
Comment (SergeyTsimfer) [inline batchflow/models/torch/layers/modules.py:48]: Previously, it was `p - layout - b`

File: batchflow/models/torch/layers/modules.py [hunk 2]
```diff
@@ -93,10 +89,10 @@ def __init__(self, inputs=None, layout='cna', filters='same', kernel_size=3,
         super().__init__()
 
         modules = nn.ModuleList()
-        global_pooling = ConvBlock(inputs=inputs, layout='V>cna', filters=filters, kernel_size=1,
-                                   dim=get_num_dims(inputs))
+        global_pooling = ConvBlock(inputs=inputs, layout='V>cnab', filters=filters,
+                                   kernel_size=1, dim=get_num_dims(inputs),
+                                   factor=None, shape=get_shape(inputs)[2:], **kwargs)
         modules.append(global_pooling)
-
```
Comment (SergeyTsimfer) [inline batchflow/models/torch/layers/modules.py:99]: Newlines are good for readability

**Review by SergeyTsimfer — APPROVED** [review]

File: batchflow/models/torch/layers/modules.py (hunk 1 above)
Comment (RisaMagpie) [reply batchflow/models/torch/layers/modules.py]: fixed

File: batchflow/models/torch/layers/modules.py (hunk 2 above)
Comment (RisaMagpie) [reply batchflow/models/torch/layers/modules.py:99]: added

**Review by SergeyTsimfer — APPROVED** [review]
Write meaningful commit messages: `fixes` does not explain anyting

**Pull request merged**